ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2070,11 +2069,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = BetterSocial/BetterSocial.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 301;
-				DEVELOPMENT_TEAM = J22M3M9VYU;
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = J22M3M9VYU;
+				DEVELOPMENT_TEAM = "";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `DEVELOPMENT_TEAM` and `CODE_SIGN_STYLE` have been changed. This could lead to issues with code signing and app distribution. Ensure that these changes are intentional and correctly managed in your CI/CD pipeline or manually before distribution.
```diff
- CODE_SIGN_STYLE = Manual;
- DEVELOPMENT_TEAM = J22M3M9VYU;
+ CODE_SIGN_STYLE = Automatic;
+ DEVELOPMENT_TEAM = "";
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->